docs/physics/update_and_conv/update_and_conv.ipynb

```diff
@@ -0,0 +1,588 @@
+{
```

Comment:
<p>Add the <code>0</code> before the decimal <code>.4</code> and <code>.45</code> for clarity   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2108/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>